2019-05-27 23:32:29.647008: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: Tesla K40c major: 3 minor: 5 memoryClockRate(GHz): 0.745
pciBusID: 0000:84:00.0
totalMemory: 11.17GiB freeMemory: 10.20GiB
2019-05-27 23:32:29.647054: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-05-27 23:32:35.241254: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-27 23:32:35.241326: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-05-27 23:32:35.241339: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-05-27 23:32:35.241773: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 9814 MB memory) -> physical GPU (device: 0, name: Tesla K40c, pci bus id: 0000:84:00.0, compute capability: 3.5)
{   'P': 'arcus_gpu.p',
    'alg': 'random',
    'alpha': 0.010139907986489696,
    'batch_norm': 0,
    'batch_size': 2048,
    'data': 'fluxbypass_aqua',
    'data_dir': '/baldig/chemistry/earth_system_science/',
    'dropout': 0,
    'env': '/home/jott1/Projects/SHERPA_EX/.profile',
    'epochs': 25,
    'gpus': '0,2,3',
    'l': "hostname='(arcus-1|arcus-2|arcus-3|arcus-4|arcus-5|arcus-6|arcus-7|arcus-8|arcus-9|arcus-10)'",
    'layer_0': 512,
    'layer_1': 512,
    'layer_2': 512,
    'layer_3': 512,
    'layer_4': 512,
    'layer_5': 512,
    'layer_6': 512,
    'leaky_relu': 0.3,
    'loss_type': 'weak_loss',
    'lr': 0.001,
    'max_concurrent': 15,
    'max_dense_layers': 7,
    'net_type': 'conservation',
    'num_layers': 5,
    'patience': 10,
    'q': 'arcus.q',
    'sch': 'local'}
Epoch 1/25
 - 7626s - loss: 247.2646 - mean_squared_error: 249.7975 - val_loss: 188.8845 - val_mean_squared_error: 190.8194
Epoch 2/25
 - 7764s - loss: 188.5579 - mean_squared_error: 190.4894 - val_loss: 179.8157 - val_mean_squared_error: 181.6577
Epoch 3/25
 - 7821s - loss: 177.9229 - mean_squared_error: 179.7455 - val_loss: 173.8084 - val_mean_squared_error: 175.5889
Epoch 4/25
 - 7591s - loss: 172.1034 - mean_squared_error: 173.8664 - val_loss: 169.8518 - val_mean_squared_error: 171.5917
Epoch 5/25
 - 8004s - loss: 168.1440 - mean_squared_error: 169.8665 - val_loss: 163.0557 - val_mean_squared_error: 164.7260
Epoch 6/25
 - 7529s - loss: 166.0153 - mean_squared_error: 167.7160 - val_loss: 171.3760 - val_mean_squared_error: 173.1315
Epoch 7/25
 - 7504s - loss: 163.9518 - mean_squared_error: 165.6313 - val_loss: 164.6373 - val_mean_squared_error: 166.3238
Epoch 8/25
 - 7703s - loss: 164.5371 - mean_squared_error: 166.2225 - val_loss: 180.7065 - val_mean_squared_error: 182.5576
Epoch 9/25
 - 7374s - loss: 161.9125 - mean_squared_error: 163.5711 - val_loss: 157.8231 - val_mean_squared_error: 159.4398
Epoch 10/25
 - 7252s - loss: 162.9927 - mean_squared_error: 164.6623 - val_loss: 159.7452 - val_mean_squared_error: 161.3816
Epoch 11/25
 - 7503s - loss: 163.8724 - mean_squared_error: 165.5511 - val_loss: 156.5607 - val_mean_squared_error: 158.1645
Epoch 12/25
 - 7568s - loss: 170.0141 - mean_squared_error: 171.7556 - val_loss: 156.5072 - val_mean_squared_error: 158.1104
Epoch 13/25
 - 7263s - loss: 159.2247 - mean_squared_error: 160.8558 - val_loss: 154.8302 - val_mean_squared_error: 156.4163
Epoch 14/25
 - 7046s - loss: 157.7150 - mean_squared_error: 159.3306 - val_loss: 160.9608 - val_mean_squared_error: 162.6097
Epoch 15/25
 - 7397s - loss: 157.5089 - mean_squared_error: 159.1224 - val_loss: 158.7510 - val_mean_squared_error: 160.3772
Epoch 16/25
 - 7290s - loss: 159.3326 - mean_squared_error: 160.9648 - val_loss: 156.9735 - val_mean_squared_error: 158.5815
/pkg/python/3.6.1-centos7/lib/python3.6/site-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Using TensorFlow backend.
Epoch 17/25
Traceback (most recent call last):
  File "main.py", line 121, in <module>
    net.train(train_gen, valid_gen, trial=trial, client=client)
  File "/home/jott1/Projects/CBRAIN-CAM/notebooks/tbeucler_devlog/hp_opt_conservation/model.py", line 166, in train
    )
  File "/home/jott1/.local/lib/python3.6/site-packages/tensorflow/python/keras/engine/training.py", line 1779, in fit_generator
    initial_epoch=initial_epoch)
  File "/home/jott1/.local/lib/python3.6/site-packages/tensorflow/python/keras/engine/training_generator.py", line 204, in fit_generator
    x, y, sample_weight=sample_weight, class_weight=class_weight)
  File "/home/jott1/.local/lib/python3.6/site-packages/tensorflow/python/keras/engine/training.py", line 1552, in train_on_batch
    outputs = self.train_function(ins)
  File "/home/jott1/.local/lib/python3.6/site-packages/tensorflow/python/keras/backend.py", line 2914, in __call__
    fetched = self._callable_fn(*array_vals)
  File "/home/jott1/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1382, in __call__
    run_metadata_ptr)
KeyboardInterrupt
